.github/workflows/github-pages.yml

```
@@ -1,18 +1,15 @@
 on:
   workflow_dispatch:
-    inputs:
-      opt-in-to-sparse-registry:
-        description: Enable/disable CARGO_UNSTABLE_SPARSE_REGISTRY (https://blog.rust-lang.org/2022/06/22/sparse-registry-testing.html)
-        required: true
-        type: boolean
-        default: false
   push:
     branches: [main]
     paths:
     - design/**
 
 name: Update GitHub Pages
 
+env:
+  rust_version: 1.66.1
```

It's not going to hurt anything to preemptively up this one
```suggestion
  rust_version: 1.67.1
```